frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { Sighting } from "../../clients/apiClient";
+import "./BrowseSightings.scss";
+
+interface SightingProps {
+  sighting: Sighting;
+}
+
+export const SightingCard: React.FunctionComponent<SightingProps> = ({
+  sighting,
+}) => {
+  return (
+    <div className="sighting-card">
+      <h3 className="fade-in">{sighting.species.name}</h3>
+
+      {sighting.imageUrl !== "" ? (
+        <img
+          className="image fade-in"
+          src={sighting.imageUrl}
+          alt={sighting.species.name}
+        />
+      ) : (
+        <>
+          <img
+            className="image fade-in"
+            src="https://clipartmag.com/images/baby-whale-clipart-19.png"
+            alt={sighting.species.name}
+          />
+          <p className="warning">No image found</p>
+        </>
```

Comment:
This is pretty bad UX! It's not an error if there isn't an image associated with the sighting, it just means they didn't upload one, but we don't require it! Just display nothing if there's no image
```suggestion
        <></>
```